features/rt/hack/deploy.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
This script needs to be executable.
```$ make deploy
./hack/deploy.sh
make: execvp: ./hack/deploy.sh: Permission denied
make: *** [Makefile:10: deploy] Error 127
```

Reply:
fixed by moving the mcp to its own feature directory